fastlane/lib/fastlane/actions/hockey.rb

```diff
@@ -26,15 +20,6 @@ def self.connection(options)
           builder.response :json, content_type: /\bjson$/
           builder.use FaradayMiddleware::FollowRedirects
           builder.adapter :net_http
-          if ENV['SPACESHIP_DEBUG']
-            # for debugging only
-            # This enables tracking of networking requests using Charles Web Proxy
-            builder.proxy "https://127.0.0.1:8888"
```

Comment:
I think this isn't a part of spaceship. I assume these might be copy and pasted from spaceship faraday code. So I delete this from hockey action.